Bottom: 3c68ae697f7c4e15e7346c72d61767c3b43fb14c
Top:    fd76b42b5be212b311ea8939bf0c79287a455cd9
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-06 09:07:03 -0500

Refresh of provision-script

---
diff --git a/provision.sh b/provision.sh
index 3762e89d..51e8abb5 100755
--- a/provision.sh
+++ b/provision.sh
@@ -9,6 +9,6 @@ ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o Contr
 exec ansible-playbook \
         --connection=ssh \
         --timeout=30 \
-        --inventory-file=ansible-playbook/staging \
+        --inventory-file=ansible-playbook/production \
         -vv \
         ansible-playbook/site.yml "$@"
